src/locale/de/_lib/formatRelative/index.ts

```diff
@@ -0,0 +1,16 @@
+import { FormatRelativeFn } from '../../../types';
+
+const formatRelativeLocale = {
+  lastWeek: "'letzten' eeee 'um' p",
+  yesterday: "'gestern um' p",
+  today: "'heute um' p",
+  tomorrow: "'morgen um' p",
+  nextWeek: "eeee 'um' p",
+  other: 'P'
+}
+
+const formatRelative: FormatRelativeFn = function formatRelative(token, _date, _baseDate, _options) {
```

Comment:
While I prefer using `function name() {...` in this case (given that we verify `FormatRelativeFn`) it's better using `() => {...`:

```suggestion
const formatRelative: FormatRelativeFn = formatRelative(token, _date, _baseDate, _options) => {
```